### PR TITLE
Agriculture resources for action

### PR DESCRIPTION
**Pull request by aabdaab**
Add Resources for Action section to Agriculture page.

image

### CODE REVIEW

File: app/javascript/app/components/sectors-agriculture/resources-for-action/resources-for-action-component.jsx [hunk 1]
```diff
@@ -1,8 +1,102 @@
...
+            </span>
+            <Button color="yellow" className={styles.button} link="">
+              NDC Funding Navigator
+            </Button>
+          </div>
+          <div className={styles.page}>
+            <div className={styles.logosContainer}>
+              {partners.map(partner => (
+                <div key={partner.title} className={styles.partnerWrapper}>
+                  <button
```
Comment (simaob) [inline app/javascript/app/components/sectors-agriculture/resources-for-action/resources-for-action-component.jsx]: Do we need to use a button here? I know that we do it in multiple places, but I find it sad that I can't copy the URL without having to open it :/

Comment (aabdaab) [reply app/javascript/app/components/sectors-agriculture/resources-for-action/resources-for-action-component.jsx]: check now 😎

File: app/javascript/app/components/sectors-agriculture/resources-for-action/resources-for-action-component.jsx [hunk 2]
```diff
@@ -1,8 +1,102 @@
...
+    img: gaezLogo,
+    link: 'http://www.fao.org/nr/gaez/en/',
+    title: 'Global Agro-Ecological Zones',
+    description:
+      'Global Ago-Ecological Zones provides data on land and water resources, agro-climatic resources, crop yields, and other pertinent data.'
+  }
+];
 
 class ResourcesForAction extends PureComponent {
+  handleLogoClick = partner => {
```
Comment (simaob) [inline app/javascript/app/components/sectors-agriculture/resources-for-action/resources-for-action-component.jsx]: so we can get rid of this? :D

Comment (aabdaab) [reply app/javascript/app/components/sectors-agriculture/resources-for-action/resources-for-action-component.jsx]: 🤦‍♀️ yeah

**Pull request merged**